packages/terra-core-docs/src/terra-dev-site/doc/pills/example/DisclosurePill.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useRef, useState } from 'react';
+import uniqueid from 'lodash.uniqueid';
+import Popup from 'terra-popup';
+import PillList, { Pill } from 'terra-pills';
+import Spacer from 'terra-spacer';
+
+const DisclosurePill = () => {
+  const [open, setOpen] = useState(false);
+  const pillRef = useRef();
+  const disclosurePillId = `terra-pills-example-disclosure-pill-${uniqueid()}`;
```

Comment:
If these examples aren't rendered more than once on a given page, you shouldn't need the uniqueid dependency. Your prefixes look unique enough already.